src/components/marsPhotoViewer/ImageFilter.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { useState, ChangeEvent } from "react";
+import "./ImageFilter.scss";
+
+export function ImageFilter() {
+
+    // const currentYear = new Date().getFullYear();
+    // const years = Array.from(new Array(20), (val, index) => currentYear - index);
+    const [selectedRover, setSelectedRover] = useState("Curiosity");
+    const [selectedCamera, setSelectedCamera] = useState("");
+    const [selectedDate, setSelectedDate] = useState("");
+    const [imageURL,setImageURL] = useState();
+
+    const handleRover = (event: ChangeEvent<HTMLSelectElement>) => {
+        setSelectedRover(event.target.value);
+    };
+
+    const handleCamera = (event: ChangeEvent<HTMLSelectElement>) => {
+        setSelectedCamera(event.target.value);
+    };
+
+    const handleDateChange = (event: ChangeEvent<HTMLInputElement>) => {
+        setSelectedDate(event.target.value);
+    };
+
+    const handleButtonClick = ()=>{
```

Comment:
We will want to write this function in the parent component and pass it in as a prop, the type would be 
`() => void`